### ISSUE TITLE
Crown copyright image does not have appropriate contrast in high contrast mode

### ISSUE THREAD

**Issue by andrewhick** [alphagov/wcag-primer#143]

**Comment by andrewhick** [comment]
Discussed with the Tech Docs team and suggest we raise this in the [tech-docs-gem repo](https://github.com/alphagov/tech-docs-gem), but suggest waiting until we've looked at the higher priority WCAG AA issues first (402, 403, 404).
